src/alembic_utils/replaceable_entity.py

```diff
@@ -173,15 +174,45 @@ def get_required_migration_op(
         return CreateOp(self)
 
 
+class EntityWithOp(NamedTuple):
+    entity: ReplaceableEntity
+    op: ReversibleOp
+
+
+def depends_on_expanded(entity: ReplaceableEntity) -> set[ReplaceableEntity]:
+    """return expanded set of objects entity depends_on including recursive deps
+    ie.: if A.depends_on=[B] and B.depends_on=[C] then depends_on_expanded(A) = {B, C}
+    """
+    expanded_deps = set()
+    for e in entity.depends_on:
+        expanded_deps.add(e)
+        expanded_deps.update(depends_on_expanded(e))
+
+    return expanded_deps
+
+
+def dependants(entities: list[ReplaceableEntity], known_obj_with_deps: list[ReplaceableEntity]) -> set[ReplaceableEntity]:
```

Comment:
Same as in case of `depends_on_expanded`, dedicated ut would be great

Reply:
added